### PR TITLE
add sync customer command and drop subscription customer constraint

### PR DESCRIPTION
**Pull request by anamarn**
**TLDR:**
Solves (https://github.com/twentyhq/private-issues/issues/212)
Add command to sync customer data from stripe to BillingCustomerTable for all active workspaces. Drop foreign key  contraint on billingCustomer in BillingSubscription (in order to not break the DB).

**In order to test:**

- Billing should be enabled
- Have some workspaces that are active and whose id's are not mentioned in BillingCustomer (but the customer are present in stripe).

Run the command: 
`npx nx run twenty-server:command billing:sync-customer-data`

Take into consideration
Due that all the previous subscriptions in Stripe have the workspaceId in their metadata, we use that information as source of true for the data sync

**Things to do:**

- Add tests for Billing utils
- Separate StripeService into multipleServices (stripeSubscriptionService, stripePriceService etc) perhaps add them in (https://github.com/twentyhq/private-issues/issues/201)?

### CODE REVIEW

File: packages/twenty-server/src/database/typeorm/core/migrations/billing/1734532875877-dropSubscriptionCustomerConstraint.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DropSubscriptionCustomerConstraint1734532875877
+  implements MigrationInterface
+{
+  name = 'DropSubscriptionCustomerConstraint1734532875877';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "core"."billingSubscription" DROP CONSTRAINT "FK_9120b7586c3471463480b58d20a"`,
```
Comment (charlesBochet) [inline packages/twenty-server/src/database/typeorm/core/migrations/billing/1734532875877-dropSubscriptionCustomerConstraint.ts:10]: this should be removed from the original migration, otherwise we will apply it first before removing it and end up in the same issue while applying it

**Review by charlesBochet — APPROVED** [review]
thank you, I've left an important comment :)

**Review by charlesBochet — APPROVED** [review]
